build.gradle.kts

```diff
@@ -92,6 +92,10 @@ sonar {
                 "dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/sme/SmeDataModel.ts," +
                 "dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/" +
+                "EuTaxonomyForNonFinancialsDataModel.ts," +
+                "dataland-frontend/src/components/forms/parts/elements/derived/ActivityTree.ts," +
```

Comment:
I found an odd bug: uploading this 
![grafik](https://github.com/d-fine/Dataland/assets/71646830/af012c88-9680-499b-928a-0aa6a411b4ed)

leads to this on the view page:

![grafik](https://github.com/d-fine/Dataland/assets/71646830/d8aed450-8d16-49bb-8f84-bdce2691540b)

the data in the backend also show these values

Reply:
On the first look it seems like a weird rounding bug. Will look into this

Reply:
This looks like a wrong "precision" in the rounding function.  1 2 3 and 4 are rounded to "0", and 5 6 7 8 are rounded to "10".
Adjusting the precision should do the job

Reply:
Should be fixed.